spec.bs

```diff
@@ -185,13 +185,19 @@ partial interface Navigator {
 
 dictionary AuctionAd {
   required USVString renderURL;
+  USVString sizeGroup;

```

Comment:
No particular reason to use USVStrings; these are treated as opaque identifiers. Fixed.

Reply:
but the [implementation](https://crsrc.org/c/third_party/blink/renderer/modules/ad_auction/auction_ad_interest_group.idl;drc=216e4cca368b91973935d2b149df3a245620baad;l=14) uses USVString. We need to keep the spec in sync with the implementation. If we want to change it to DOMString, we need to change the implementation as well, which I hope don't need to request some approvals first.

Reply:
I agree these should match. If we are not planning on changing the implementation, please revert my suggestion here. Although I would prefer we update the implementation to match the spec.

Reply:
Reverted

Reply:
I filed a ticket somewhere to update the implementation

Reply:
Note: Currently, the IDL fields sizeGroup in AuctionAd and width/height AuctionAdInterestGroupSize are specified to use USVString rather than DOMString, only because the Chromium implementation currently does. This contradicts the recommended use of DOMString for anything that won't appear in a UI. This issue is tracked at https://github.com/WICG/turtledove/issues/1250

Reply:
I added a comment to the spec ad I discussed previously with @domfarolino.  We should continue discussion on #1250 and can update the spec and impl as necessary.